db/migrate/20230731090232_add_fields_to_accounts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddFieldsToAccounts < ActiveRecord::Migration[6.1]
+  def change
+    change_table :accounts, bulk: true do |t|
+      t.string :domain
+      t.string :organization
```

Comment:
Az issue leírásából elsőre nekem is ködös volt. Az én olvasatomban, nincs szükség `:organisation` mezőre, hanem ez egyenértékű a `belongs_to :project` kapcsolattal, ami már definiálva van a modellen. Kérlek járj utána az ügyfélnél, hogy mi a pontos elvárás!